### PR TITLE
Fixed submit scripts for data

### PR DESCRIPTION
**Pull request by marco-link**
tag with ~~`2022-03-01_v6` `2022-03-01_v7`, wait for #6 to be resolved.~~
Let's skip tagging.

### CODE REVIEW

File: crab/submit.py [hunk 1]
```diff
@@ -73,9 +75,9 @@ def verifySample(sample):
 config.section_("JobType")
 config.JobType.pluginName = 'Analysis'
 config.JobType.psetName = '../NANOProducer/test/produceNANO.py'
-config.JobType.scriptArgs = [isData, 'year={}'.format(args.year)]
+config.JobType.outputFiles = ['nanox.root']
 #config.JobType.inputFiles = ['']
-config.JobType.maxMemoryMB = 2500
+config.JobType.maxMemoryMB = 2000
```
Comment (matt-komm) [inline crab/submit.py]: note that the default limit is 2500 MB; so is there a need to lower this?

File: crab/submit.py [hunk 2]
```diff
@@ -85,9 +87,20 @@ def verifySample(sample):
 config.Data.splitting = 'FileBased'
```
Comment (matt-komm) [inline crab/submit.py:88]: if you run over data with lumi mask, do you need to use LumiBased here instead?

File: crab/submit.py [hunk 3]
```diff
@@ -113,6 +126,16 @@ def verifySample(sample):
                 continue
 
             requestName = sample.split('/')[1]
+            if args.data:
+                requestName = sample.split('-')[0].replace('/', '_')[1:]
+
+            # LHE container not available for diboson samples
+            addSignalLHE = 'addSignalLHE=1'
```
Comment (matt-komm) [inline crab/submit.py:133]: better to use 'True' and 'False' instead of 0 and 1 since it is a boolean?

File: crab/submit.py (hunk 1 above)
Comment (marco-link) [reply crab/submit.py]: Not really, crab complained about not using more than 70% of memory.

File: crab/submit.py (hunk 2 above)
Comment (marco-link) [reply crab/submit.py:88]: According to the twiki `LumiBased` is prefered for data, but I had bad experience with merging multiple files.
If there is a problem with one file all other fail as well and splitting cannot be changed afterwards.

File: crab/submit.py (hunk 3 above)
Comment (marco-link) [reply crab/submit.py:133]: No, it works with integers.
I am not sure the python2 parser can interpret the string `True`.

**Pull request merged**